orm/src/test/java/org/hibernate/search/test/engine/NumericFieldTest.java

```diff
@@ -205,29 +211,29 @@ public void testByteDocumentIdMappedAsStringFieldByDefault() {
 	@Test
 	@TestForIssue(jiraKey = "HSEARCH-1987")
 	public void testOneOfSeveralFieldsIsNumeric() {
-		try ( Session session = openSession() ) {
-			FullTextSession fullTextSession = Search.getFullTextSession( session );
-			Transaction tx = fullTextSession.beginTransaction();
-
-			QueryContextBuilder queryBuilder = fullTextSession.getSearchFactory().buildQueryBuilder();
-			Query query = queryBuilder.forEntity( TouristAttraction.class ).get().all().createQuery();
-
-			@SuppressWarnings("unchecked")
-			List<Object[]> list = fullTextSession.createFullTextQuery( query, TouristAttraction.class )
-					.setProjection( ProjectionConstants.DOCUMENT )
-					.list();
-
-			assertEquals( 1, list.size() );
-			Document document = (Document) list.iterator().next()[0];
-
-			IndexableField scoreNumeric = document.getField( "scoreNumeric" );
-			assertThat( scoreNumeric.numericValue() ).isEqualTo( 23 );
+		assertEquals(
```

Comment:
I am not sure about removing these tests. Do we test elsewhere that the indexed fields are effectively numeric? If not, I would add your new test for Es/Lucene agnostic tests that the field bridge is OK but I think I would create backend specific tests to check that the indexed document is OK.